.idea/MyHashSet.java

```diff
@@ -0,0 +1,32 @@
+/*
+Intuition : as this is limited number  (10 ^ 6), we can use boolean arrays
```

Comment:
This is not the optimal solution. You created an array of 1M buckets which is wastage of space.